spec/spec.ddoc

```diff
@@ -30,6 +30,12 @@ $(T style,
         counter-increment: p;
         content: counter(p) ". ";
     }
+    h1::before, h2::before, h3::before, h4::before, p::before
+    {
+        color: #999;
+        font-size: 80%;
```

Comment:
I'm no visual expert or designer, but I'm not so sure whether reducing the font looks good.